server/lib/src/repl/tests.rs

```diff
@@ -1881,6 +1878,143 @@ async fn test_repl_increment_consumer_ruv_trim_past_valid(
     drop(server_b_txn);
 }
 
+// Test two synchronised nodes where changes no are occuring - this situation would previously
```

Comment:
Is [no how you make repl](https://youtu.be/WADdeYZYmrE?t=4)?

```suggestion
// Test two synchronised nodes where changes are not occuring - this situation would previously
```